package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "com.akbiggs.unitytimer",
+  "version": "1.1.0",
+  "displayName": "Unity Timer",
+  "description": "Run actions after a delay in Unity3D.",
+  "unity": "2018.1",
+ "keywords": [
+    "timer",
+    "coroutine",
+    "callback"
+  ],
+  "author": {
+    "name": "Alexander Biggs and Adam Robinson-Yu",
```

Comment:
Are you comfortable if we give you a shout-out here? e.g. add a "contributors" field next to "author" with yourself listed, based on the documentation here https://flaviocopes.com/package-json/#contributors 

we'll be maintaining the UPM support, I just think it would be nice to call out what you did if you're okay with it :)

Reply:
I'd be honoured! Will update this tomorrow. Do you want me to adjust the listed version number while I'm making changes?

Reply:
Nah bumping the version number is fine, it keeps the people excited